oauth/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "librespot-oauth"
+version = "0.5.0-dev"
+rust-version = "1.73"
+authors = ["Paul Lietar <paul@lietar.net>"]
+description = "Spotify OAuth"
```

Comment:
Following the other crates, for consistency maybe a bit expanded like "The OAuth logic for username and password authentication with Spotify"